package/src/prmon.cpp

```diff
@@ -260,6 +270,8 @@ int main(int argc, char* argv[]) {
         << default_json_summary << ")\n"
         << "[--interval, -i TIME]     Seconds between samples (default "
         << default_interval << ")\n"
+        << "[--store-hw-info, -s]     Store hardware information (default "
+        << default_store_hw_info << ")\n"
```

Comment:
This prints a `0` which isn't really nice. How about changing to
```
<< default_store_hw_info ? "true" : "false" << ...
```

Reply:
Good point, updated.